src/components/TextItem.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+
+export interface TextProps {
+    textTitle: string,
+}
+
+class TextItem extends React.Component<TextProps> {
```

Comment:
I don't really get why we need this class. It doesn't even have any styling. The only thing special about this class is that it uses [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label), but <label> is supposed to be used with `<input>`, and we're literally just writing some strings. Isn't this an unnecessary complication?

Reply:
Yeah sure. It used to be a bigger component with multiple tags in it, and while breaking it down this became nothing more than a standalone label tag.

Reply:
So why do we need this? It's like creating an object that only contains a string.

Reply:
I am going to remove this and use plain html tags.

Reply:
👌